config/pipeline.yaml

```diff
@@ -1825,6 +1825,135 @@ scheduler:
     platforms:
       - qemu-x86
 
+  - job: rt-tests-cyclicdeadline
+    event: &kbuild-gcc-12-arm64-preempt_rt-node-event
+      channel: node
+      name: kbuild-gcc-12-arm64-preempt_rt
+      result: pass
+    runtime: *runtime-lava-collabora
+    platforms: &kbuild-gcc-12-arm64-preempt_rt-platforms
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8186-corsola-steelix-sku131072
+      - sc7180-trogdor-kingoftown
+
+  - job: rt-tests-cyclicdeadline
+    event: &kbuild-gcc-12-arm64-preempt_rt_chromebook-node-event
+      <<: *kbuild-gcc-12-arm64-preempt_rt-node-event
+      name: kbuild-gcc-12-arm64-preempt_rt_chromebook
+    runtime: *runtime-lava-collabora
+    platforms: &kbuild-gcc-12-arm64-preempt_rt_chromebook-platforms
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+      - sc7180-trogdor-lazor-limozeen
+
+  - job: rt-tests-cyclicdeadline
+    event: &kbuild-gcc-12-x86-preempt_rt-node-event
+      channel: node
+      name: kbuild-gcc-12-x86-preempt_rt
+      result: pass
+    runtime: *runtime-lava-collabora
+    platforms: &kbuild-gcc-12-x86-preempt_rt-platforms
+      - acer-cb317-1h-c3z6-dedede
+      - acer-cbv514-1h-34uz-brya
+      - acer-R721T-grunt
+      - acer-cp514-3wh-r0qs-guybrush
```

Comment:
Same question about not using a chromebook kbuild for those.

Reply:
Fixed now.